### ISSUE TITLE
Can't manage users

### ISSUE THREAD

**Issue by dosch** [idno/known#2788]
In the site configuration I tried to manage the current users with an account. But the window stays empty.

![Screenshot 2020-07-02 at 21 23 50](https://user-images.githubusercontent.com/5372610/86401393-6a2cbb00-bcaa-11ea-9c79-acacf9f0f53c.png)

I am on shared hosting with version `mapkyca-known-1.2.2-2020061101`

**Comment by johanbove** [comment]
I am probably experiencing similar issues. There are some JavaScript error messages which are shown in the browser's console window (ctrl+shift+i). Do you see errors as well?

<img width="838" alt="Annotation 2020-07-02 214703" src="https://user-images.githubusercontent.com/922765/86403252-a01f6e80-bcad-11ea-9f72-c021ac22d9be.png">

**Comment by mapkyca** [comment]
@johanbove Well, that looks like a server config issue - css & js being served as html, which points to a mime config error.

**Comment by mapkyca** [comment]
@dosch A console log would be useful

**Comment by johanbove** [comment]
@mapkyca Thanks for the quick reply. Yeah, it doesn't seem related. My deploy of the last Known version was incomplete.

**Comment by dosch** [comment]
thnx for the fast answer. You mean this console log?

![Screenshot 2020-07-02 at 22 14 04](https://user-images.githubusercontent.com/5372610/86405280-65b7d080-bcb1-11ea-9c66-a8d1faeb347c.png)

**Comment by johanbove** [comment]
@mapkyca I think I see what @dosch  is referring to. Please see my first reply.

**Comment by mapkyca** [comment]
Hmm... pretty sure I already fixed that. Try updating from git.

**Comment by mapkyca** [comment]
Ya, I think I fixed that. I'll try and put a new build together today...

**Comment by dosch** [comment]
So, I wait for a new build from you, right? 
Because, as far as I understand, the version on github is older then the version on your website... right?

**Comment by mapkyca** [comment]
Na, GitHub is always bleeding edge (but slightly more complex to use as you need to run `composer install` yourself)

**Comment by dosch** [comment]
ah yeah; that is sadly not possible on my shared hosting environment...

**Comment by johanbove** [comment]
@mapkyca  Thanks for the new build release! Updating now.

**Comment by davidc** [comment]
If anyone else is coming here with the same problem (not fixed in the current release 1.2.2)

in `Idno/Pages/Search/User.php` change

```
            $t = new \Idno\Core\Template();
```

to

```
            $t = new \Idno\Core\DefaultTemplate();
```

see commit fa39c5a13067c517badf5f1b16c079b797414740
